help/access/file-shares.md

```diff
@@ -1,302 +1,237 @@
-   * _Can edit_: Users can rename and edit
-   * _Can delete_: Users can delete
-   * _Can share_: Users can reshare with any other user or group.<!--how does it work with URL links? -->
+## File Manager
 
-   > Note: For each _file share_, Datagrok automatically assigns a _friendly name_ (a name displayed in the UI) that
-   > specifies all the directory names starting from the root folder. While you _can_ change the _friendly name_, we
-   > encourage you to exercise caution. Changing the item's _friendly name_ also changes its unique
-   > _namespace-qualified name_ used in URL links, automation workflows, and more.
+The **File Manager** is an interface that allows you to manage connections, browse and preview file content, and perform standard file and folder actions such as opening, downloading, deleting, and renaming. To access an object's context actions, right-click it or left-click and expand the **Actions** pane in the **Context Panel** on the left. By clicking a file or folder in the **File Manager**, you can open its preview. Double-clicking a file opens it in Datagrok, and double-clicking a folder expands its content.
 
-4. Optionally, enter a description in the text field provided. You may also notify the users you share with. If you
-   don’t want to send a notification, uncheck the **Send notification** checkbox.
+:::note
 
-   > Note: To send an email notification, enter the user's email in the identity/email field. The email notification
-   > contains a link to the shared item and entered description. If you enter a user or group name, they will be
-   > notified via the Datagrok interface.
+If you don't see a certain action, it may be due to insufficient permissions. For files and folders shared with you, contact the credentials owner. If you are a credentials owner, contact the data source owner.
 
-5. Click **OK** to share. Once shared, the shared item appears in the recipient's **Folder Tree**.
+:::
 
-   ![Share a folder](share-the-folder.gif)
+In addition to the hierarchical browsing, the **File Manager** offers an array of advanced preview and data augmentation capabilities using **Directory**, **Preview**, and **Context Panel**.
 
-   > Note: When you share a file or a folder, Datagrok automatically indexes folders <!--link out to indexing when ready-->
-   > and working--> and extracts basic metadata (like the date created or its size). File indexing is optional (to index
-   > files, toggle **Index Files** in the **Share...** dialog).
+The **Directory** section shows the contents of your current folder with three viewing modes: icons, cards, and grid. You can click an object to view its content in the **Preview**, or right-click it to access available context actions. You can also use the search bar located above the **Directory** to search for files and folders within your current directory. The search bar allows you to search for items by name, file extension, or metadata.
```

Comment:
The **Directory** section shows the contents of your current folder with three viewing modes: icons, cards, and grid
=>
Drop everything after "folder". Icons are not important here. You can mention it later

Reply:
"You can click an object to view its content in the **Preview**, or right-click it to access available context actions"
=>
Click on a file to see its preview and properties, or right-click for more options"

Reply:
You can also use the search bar located above the **Directory** to search for files and folders within your current directory
=>
Drop "you can also". Drop "located above the directory"